src/snowflakes/viewconfigs/views.py

```diff
@@ -0,0 +1,41 @@
+from pyramid.view import view_config
+from snovault.viewconfigs.report import ReportView
+from snovault import AbstractCollection 
+from snovault.resource_views import collection_view_listing_db
+from snovault.viewconfigs.searchview import SearchView
+
+def includeme(config):
+    config.add_route('search', '/search{slash:/?}')
+    config.add_route('report', '/report{slash:/?}')
+    config.scan(__name__)
+
+
+DEFAULT_DOC_TYPES = [
```

Comment:
Move this variable above 'def includeme'.  There should be two spaces between imports block, variables block and then functions.